cmd/otel-allocator/README.md

```diff
@@ -189,6 +189,29 @@ The CRs can be filtered by labels as documented here: [api/opentelemetrycollecto
 
 Upstream documentation here: [PrometheusReceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver#opentelemetry-operator)
 
+### Pod/Service Monitor Selectors
+
+As of `v1beta1` of the OpenTelemetryOperator, a `serviceMonitorSelector` and `podMonitorSelector` must be included, even if you don’t intend to use it, like this:
+
+```yaml
+prometheusCR:
+  enabled: true
+  podMonitorSelector: {}
+  serviceMonitorSelector: {}
+```
+
+This will make the TargetAllocator scrap all the Service and Pod Monitors inside of the cluster. If you need something more specific, you can also add a label filter:
```

Comment:
```suggestion
This will make the TargetAllocator scrape all the Service and Pod Monitors inside of the cluster. If you need something more specific, you can also add a label filter:
```